_build/resolvers/policies.resolver.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/** @var modX $modx */
+$modx =& $object->xpdo;
```

Comment:
Have you tested installing a build? 

Reason I'm asking is I think this may need to be $transport->xpdo because the resolver is added directly to the package (`$builder->package->put(..)`) rather than to a vehicle. It's possible CB has that differently.

Reply:
Hey! Yeah tested and working nicely. 
There was a problem with the policy not being applied to the admin group the first time, but working on a reinstall. I fixed that by reordering those, making sure the policy actually existed before attempting to assign it.
Hmmm.. now you come to mention it though, there was a PHP warning in the install log but I thought that was related to the guzzle package. I'll double check that in the morning.